content/en/docs/instrumentation/php/automatic.md

```diff
@@ -49,6 +49,13 @@ function ends it.
 
     * **pickle** can be used to install extensions that are available via http://pecl.php.net, however that's not the case
     for opentelemetry-php-instrumentation yet, so the only way for it is to install directly from source code.
+
+        **Installing from source requires proper development environment and few dependendencies:**
+
+        - `C compiler`
+        - `autoconf`
+        - `make`
```

Comment:
would it be possible to provide some specific examples? For C++ we have tabs that give examples for some linux distros and macos:

https://opentelemetry.io/docs/instrumentation/cpp/getting-started/

Reply:
ok, thx. Will try to provide something similar.

Reply:
TY